src/infrastructure/content/ServerPort.java

```diff
@@ -0,0 +1,11 @@
+package infrastructure.content;
+
+public class ServerPort implements IServerPort {
+	/*
+	 * This method will save the port of the board server locally to send message over networking
+	 */
+	@Override
+	public void sendPort(int port) {
```

Comment:
Check if @Override allows empty definition. If it doesn't allow it, give return for void functions.

Reply:
Usually, eclipse shows error for such scenario. There was no error in it.